XCTest.xcodeproj/project.pbxproj

```diff
@@ -255,6 +299,24 @@
 /* End PBXLegacyTarget section */
 
 /* Begin PBXNativeTarget section */
+		19D66F931DE6F57E00C112C0 /* SwiftXCTest-iOS */ = {
```

Comment:
Well, SwiftPM doesn't build from Xcode projects, so there's be no effect there. (It would be interesting to see sometime though if we could do away with this Xcode project altogether and just let SwiftPM generate it for us!)

I'm not sure about Carthage, though I would consider it to be their bug if it doesn't work, since xcodebuild deals with that setup just fine.